trymito.io/pages/industries/financial-services.tsx

```diff
@@ -0,0 +1,227 @@
+                    </section>
+                    <section className={pageStyles.gradient_card}>
+                        <div className={classNames(pageStyles.subsection, 'center')}>
+                        <h2 className={classNames('margin-top-4rem', 'margin-bottom-2rem')}>
+                            Spreadsheet Automation Use Cases
+                        </h2>
+                        </div>
+                        <div className={pageStyles.subsection}>
+                            <div className={securityStyles.security_bullet_container}>
+                                <div className={securityStyles.icon}>
+                                    <Image className={securityStyles.icon} src={ReconIcon} alt='icon'></Image>
+                                </div>
+                                <h3>
+                                    Reconciliation
+                                </h3>
+                                <p>
+                                    Performing end of quarter data reconciliation processes in Excel pushes back quarter close by up to 15 days.
+                                </p>
+                                <p>
+                                    Mito uses Python to connect directly to your data sources and perform the recon, so you can refresh your recon in seconds not hours. 
+                                </p>
+                                <p>
+                                    Not having to wait hours to check the status of the recon means you can close the loop with your reconciliation partners faster. Close your books and get a head start on the next quarter.
+                                </p>
+                            </div>
+                            <div className={classNames(securityStyles.security_bullet_container, pageStyles.subsection_second_element_mobile_spacing)}>
+                                <div className={securityStyles.icon}>
+                                    <Image className={securityStyles.icon} src={ReportIcon} alt='icon'></Image>
+                                </div>
+                                <h3>
+                                    Reporting
+                                </h3>
+                                <p>
+                                    Building reports for management is never one and done. They&apos;re always going to want you pull the most recent data and update the formatting of tables. 
+                                </p>
+                                <p>
+                                    Updating your report in Mito is as easy as clicking “Run All”. It will pull the most up to date data, refresh your analysis, and spit out the updated Excel file. 
+                                </p>
+                                <p>
+                                    Refreshing a report that quickly means you can build a report 10 minutes before a meeting with the most up to date data instead of building it the night before. 
```

Comment:
```suggestion
                                    Stop delivering out of date data. Mito-generated Python scripts can be used to build a report 10 minutes before a meeting with the most up to date data, instead of staying up late manually building a report the night before. 
```